setup-database.sh

```diff
@@ -60,11 +63,13 @@ for db in $(echo ${POSTGRES_DBNAME} | tr ',' ' '); do
             echo "Loading legacy sql"
             su - postgres -c "psql ${db} -f ${SQLDIR}/legacy_minimal.sql" || true
             su - postgres -c "psql ${db} -f ${SQLDIR}/legacy_gist.sql" || true
+            export PGPASSWORD=${POSTGRES_PASS}
+            psql ${db} -U ${POSTGRES_USER} -p 5432 -h localhost -f custom.sql
```

Comment:
postgres user or replication user?

Reply:
Postgres users because of the permissions it has.